pcww/builder
# Modal Systems Implementation

## [Issue](https://github.com/pcww/builder/issues/66)

## Objectives
Implement a modal system that allows you to view images in a larger view. This will be a pattern that can be implemented for other thumbnails, example boards, etc. 

Ideally, you will be able to click a thumbnail and a modal pops up with larger image and caption with ability to next/previous between available images. 

## Setup
- This adds react-modal. Run `npm install`

## Testing
- This is currently being implemented for the endcaps
1. Open the builder
1. Go to step 2 in the wizard
1. Click on the thumbnail for the endcap modal to pop up

_Testing next/previous_
Next: Clicking on the image and the right button should go to the next image
Previous: Clicking on the left button should go to the previous image

_Close_
Clicking on the top right button should close the modal
